<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>320</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_clear">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:rgb(183, 140, 255);
	border: 1px solid rgb(0, 0, 0);
    text-color:rgb(0, 0, 0);
}
QPushButton:pressed{
	background-color: rgb(164, 126, 232);
	border: 1px solid rgb(0, 0, 0);
}</string>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_change">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>90</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:rgb(183, 140, 255);
	border: 1px solid rgb(0, 0, 0);
    text-color:rgb(0, 0, 0);
}
QPushButton:pressed{
	background-color: rgb(164, 126, 232);
	border: 1px solid rgb(0, 0, 0);
}</string>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sin">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>90</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:rgb(183, 140, 255);
	border: 1px solid rgb(0, 0, 0);
    text-color:rgb(0, 0, 0);
}
QPushButton:pressed{
	background-color: rgb(164, 126, 232);
	border: 1px solid rgb(0, 0, 0);
}</string>
    </property>
    <property name="text">
     <string>√</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_divide">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>90</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:rgb(183, 140, 255);
	border: 1px solid rgb(0, 0, 0);
    text-color:rgb(0, 0, 0);
}
QPushButton:pressed{
	background-color: rgb(164, 126, 232);
	border: 1px solid rgb(0, 0, 0);
}</string>
    </property>
    <property name="text">
     <string>÷</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_dobutok">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>160</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:rgb(183, 140, 255);
	border: 1px solid rgb(0, 0, 0);
    text-color:rgb(0, 0, 0);
}
QPushButton:pressed{
	background-color: rgb(164, 126, 232);
	border: 1px solid rgb(0, 0, 0);
}</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>160</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:rgb(184, 255, 251);
	border: 1px solid rgb(0, 0, 0);
    text-color:rgb(0, 0, 0);
}
QPushButton:pressed{
	background-color: rgb(175, 244, 240);
	border: 1px solid rgb(0, 0, 0);
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>160</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:rgb(184, 255, 251);
	border: 1px solid rgb(0, 0, 0);
    text-color:rgb(0, 0, 0);
}
QPushButton:pressed{
	background-color: rgb(175, 244, 240);
	border: 1px solid rgb(0, 0, 0);
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>160</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:rgb(184, 255, 251);
	border: 1px solid rgb(0, 0, 0);
    text-color:rgb(0, 0, 0);
}
QPushButton:pressed{
	background-color: rgb(175, 244, 240);
	border: 1px solid rgb(0, 0, 0);
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_minus">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>300</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:rgb(183, 140, 255);
	border: 1px solid rgb(0, 0, 0);
    text-color:rgb(0, 0, 0);
}
QPushButton:pressed{
	background-color: rgb(164, 126, 232);
	border: 1px solid rgb(0, 0, 0);
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>230</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:rgb(184, 255, 251);
	border: 1px solid rgb(0, 0, 0);
    text-color:rgb(0, 0, 0);
}
QPushButton:pressed{
	background-color: rgb(175, 244, 240);
	border: 1px solid rgb(0, 0, 0);
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>230</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:rgb(184, 255, 251);
	border: 1px solid rgb(0, 0, 0);
    text-color:rgb(0, 0, 0);
}
QPushButton:pressed{
	background-color: rgb(175, 244, 240);
	border: 1px solid rgb(0, 0, 0);
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>230</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:rgb(184, 255, 251);
	border: 1px solid rgb(0, 0, 0);
    text-color:rgb(0, 0, 0);
}
QPushButton:pressed{
	background-color: rgb(175, 244, 240);
	border: 1px solid rgb(0, 0, 0);
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_plus">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>230</y>
      <width>80</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:rgb(183, 140, 255);
	border: 1px solid rgb(0, 0, 0);
    text-color:rgb(0, 0, 0);
}
QPushButton:pressed{
	background-color: rgb(164, 126, 232);
	border: 1px solid rgb(0, 0, 0);
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>300</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:rgb(184, 255, 251);
	border: 1px solid rgb(0, 0, 0);
    text-color:rgb(0, 0, 0);
}
QPushButton:pressed{
	background-color: rgb(175, 244, 240);
	border: 1px solid rgb(0, 0, 0);
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>300</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:rgb(184, 255, 251);
	border: 1px solid rgb(0, 0, 0);
    text-color:rgb(0, 0, 0);
}
QPushButton:pressed{
	background-color: rgb(175, 244, 240);
	border: 1px solid rgb(0, 0, 0);
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>300</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:rgb(184, 255, 251);
	border: 1px solid rgb(0, 0, 0);
    text-color:rgb(0, 0, 0);
}
QPushButton:pressed{
	background-color: rgb(175, 244, 240);
	border: 1px solid rgb(0, 0, 0);
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_res">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>370</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
      <pointsize>24</pointsize>
      <stylestrategy>NoAntialias</stylestrategy>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:rgb(183, 140, 255);
	border: 1px solid rgb(0, 0, 0);
}
QPushButton:pressed{
	background-color: rgb(164, 126, 232);
	border: 1px solid rgb(0, 0, 0);
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_dot">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>370</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:rgb(184, 255, 251);
	border: 1px solid rgb(0, 0, 0);
    text-color:rgb(0, 0, 0);
}
QPushButton:pressed{
	background-color: rgb(175, 244, 240);
	border: 1px solid rgb(0, 0, 0);
}</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_0">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>370</y>
      <width>160</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:rgb(184, 255, 251);
	border: 1px solid rgb(0, 0, 0);
    text-color:rgb(0, 0, 0);
}
QPushButton:pressed{
	background-color: rgb(175, 244, 240);
	border: 1px solid rgb(0, 0, 0);
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>320</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	background-color:rgb(255,255,255);
    text-color:rgb(0, 0, 0);
	border: 1px solid rgb(0, 0, 0);

}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="history">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>320</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
      <pointsize>20</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	background-color:rgb(230, 228, 232);
    text-color:rgb(0, 0, 0);
  	border: 1px solid rgb(0, 0, 0);


}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
